plugins/inputs/postgresql_extensible/sample.conf

```diff
@@ -35,20 +35,30 @@
   # default, all rows inserted with current time. By setting a timestamp column,
   # the row will be inserted with that column's value.
   #
+  # The minversion field specifies minimal database version this query
+  # will run on.
+  #
+  # The maxversion field when set specifies maximal database version
+  # this query will NOT run on.
+  #
+  # Database version in `minversion` and `maxversion` is represented as
+  # a single integer without last component, for example:
+  # 9.6.2 -> 906
+  # 15.2 -> 1500
+  #
   # Structure :
   # [[inputs.postgresql_extensible.query]]
   #   sqlquery string
-  #   version string
+  #   minversion int
+  #   maxversion int
```

Comment:
By default TOML will name them `min_version` and `max_version` if I'm not mistaken...